hudi-client/hudi-flink-client/pom.xml

```diff
@@ -35,7 +35,21 @@
     </properties>
 
     <dependencies>
-        <!-- Hudi  -->
+        <!-- Logging -->
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-1.2-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
```

Comment:
nit: Indentation is off?

Reply:
This will be addressed in [another PR](https://github.com/apache/hudi/pull/6550) to save CI resource.